### PR TITLE
#38720 Admin view/categorise/download feedback page

### PR DESCRIPTION
**Pull request by jamesgriff**
Allow admins to view / download / categorise (spam / not spam) feedback

### CODE REVIEW

File: GenderPayGap.WebUI/Views/AdminFeedback/ViewFeedback.cshtml [hunk 1]
```diff
@@ -0,0 +1,266 @@
...
+        };
+    }
+
+    @(Html.GovUkBreadcrumbs(new BreadcrumbsViewModel
+    {
+        Crumbs = crumbs
+    }))
+}
+
+@{
```
Comment (aidayounesi) [inline GenderPayGap.WebUI/Views/AdminFeedback/ViewFeedback.cshtml]: Thanks James, It looks great. Can we probably break this file to two files, one partial view FeedbackRows?

File: GenderPayGap.WebUI/Views/Admin/Home.cshtml [hunk 2]
```diff
@@ -96,12 +96,19 @@
...
-    Download a CSV of feedback submitted through the "Send Feedback" links
-    <br/>
+    <a href="@Url.Action("ViewFeedback", "AdminFeedback")" class="govuk-link govuk-!-font-weight-bold">View and download feedback</a>
+    <br />
+    View and download feedback submitted through the "Send Feedback" links
+    <br />
+    @if (Model.NewFeedbackCount > 0)
+    {
+        <span class="govuk-!-font-weight-bold">
+            @(Model.NewFeedbackCount) new responses
```
Comment (raamSoftwire) [inline GenderPayGap.WebUI/Views/Admin/Home.cshtml]: This is a cute feature

File: GenderPayGap.WebUI/Views/AdminFeedback/ViewFeedback.cshtml [hunk 3]
```diff
@@ -0,0 +1,266 @@
...
+    object FeedbackRows(List<Feedback> feedbackRows, string buttonClasses)
+    {
+        Func<object, object> currentStateHtml = @<text>
+            @foreach (Feedback feedback in feedbackRows)
+            {
+                <tr class="govuk-table__row">
+                    <td class="govuk-table__cell" rowspan="7" style="border-bottom: 2px solid #b1b4b6;">
```
Comment (raamSoftwire) [inline GenderPayGap.WebUI/Views/AdminFeedback/ViewFeedback.cshtml]: It doesn't feel great having colours here. Is it worth setting up a stylesheet?

Comment (jamesgriff) [reply GenderPayGap.WebUI/Views/AdminFeedback/ViewFeedback.cshtml]: Good idea. In the end I didn't even need the colours.

File: GenderPayGap.WebUI/Views/AdminFeedback/ViewFeedback.cshtml (hunk 1 above)
Comment (jamesgriff) [reply GenderPayGap.WebUI/Views/AdminFeedback/ViewFeedback.cshtml]: Good idea - it looks much nicer now as two files

**Pull request merged**